### PR TITLE
DSP-293 Update contact information on the landing page

### PR DESCRIPTION
**Pull request by waychal**
On landing page, update contact information for:
- email address with default subject
- post address
- phone number

resolves [DSP-293](https://dasch.myjetbrains.com/youtrack/issue/DSP-293)

### CODE REVIEW

File: src/app/main/footer/footer.component.html [hunk 1]
```diff
@@ -21,13 +21,13 @@
         <div class="box contact">
             <a mat-button href="https://dasch.swiss" target="_blank">Data and Service Center for the Humanities</a>
             <a mat-button href="https://www.google.com/maps/place/Bernoullistrasse+32%2C+4056+Basel" target="_blank">
```
Comment (RalfBarkow) [inline src/app/main/footer/footer.component.html:23]: google maps link still points to Bernoullistrasse

Comment (waychal) [reply src/app/main/footer/footer.component.html]: @RalfBarkow thanks. I have corrected the links.

**Review by flavens — APPROVED** [review]
It looks good to me. 
One detail, you have to fill in the assignee and label sections of the PR - especially the label because we use it.them in the release notes (see contribution documentation). I did it for you here.

**Comment by waychal** [comment]
@flavens Thanks. I will do it from next time. I will merge the PR now.

**Pull request merged**